viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx

```diff
@@ -315,7 +317,11 @@ export class TraceExplorerWidget extends ReactWidget {
                 outputDescription = outputDescriptors[props.index].description;
             }
         }
-        return <div className='outputs-list-container' key={props.key} style={props.style} onClick={this.outputClicked.bind(this, props.index)}>
+        let traceContainerClassName = 'outputs-list-container';
+        if (props.index == this.lastSelectedOutputIndex) {
+            traceContainerClassName = traceContainerClassName + ' theia-mod-selected';
+        }
+        return <div className={traceContainerClassName} key={props.key} style={props.style} onClick={this.outputClicked.bind(this, props.index)}>
```

Comment:
I wonder if we should synchronize the graph with the selection (like link with editor in Eclipse). I miss that in Trace Compass mainline. Maybe not part of this patch.

Reply:
What do you mean by graph? See this issue here, I think it covers various use cases. That will be for another patch though. It's another issue.

https://github.com/theia-ide/theia-trace-extension/issues/73

Reply:
I meant selecting a graph (e.g. CPU usage) on the right, should it then select the corresponding analysis in the list of available analysis. Yes, that could be another issue.